agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -231,6 +231,22 @@ class backhaul_manager : public btl::transport_socket_thread {
 
     std::unique_ptr<beerocks::agent_ucc_listener> m_agent_ucc_listener;
 
+    /**
+     * @brief Type definition for associated clients information.
+     *
+     * Associated client information consists of:
+     * - The MAC address of the 802.11 client that associates to a BSS.
+     * - Timestamp of the 802.11 client's last association to this Multi-AP device.
+     *
+     * Associated client information is gathered from
+     * ACTION_BACKHAUL_CLIENT_ASSOCIATED_NOTIFICATION events received from slave threads.
+     *
+     * Associated client information is later used to fill in the Associated Clients TLV
+     * in the Topology Response message.
+     */
```

Comment:
It's really great that you write such a comprehensive description of the new container.

Reply:
Thanks